### PR TITLE
Steven Fiero.Advanced.Number 2

### PR DESCRIPTION
**Pull request by ssfiero**

### CODE REVIEW

**Comment by ssfiero** [comment]
Lesson Three - Contact List - Part 2:  Updated db.json file to match the array of objects for each contact, which now resides in the App.js file.

File: src/Action.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Log = props => {
+  // Setup is similar to Contact.js
+  return (
+    <li className="Log">
+      <div className="lg-col-3">
+        <button
+          // Button to delete action
+          onClick={() => props.onDeleteAction(props.log)}
```
Comment (canaandavis) [inline src/Action.js]: This is looking good, but I don't think you passed a prop named `onDeleteAction`, looks like you pass it as `onClick` from your `ActionLog` component

File: src/ActionLog.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Log from './Action';
+
+const ActionLog = props => {
+  // Setup is similar to ContactList.js
+  return (
+    <ul className="Action-Log">
+      {props.actionLog.map(log => {
```
Comment (canaandavis) [inline src/ActionLog.js]: `.map` actually receives two arguments, along with the element in the array it receives the `index` of that element.

So `props.actionLog.indexOf(log)` is not necessary.

The following could be used and you can pass the second argument, `index` as your `key` prop!

`props.actionLog.map( ( log, index ) => {`

File: src/ContactList.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Contact from './Contact';
+
+// Converted from a class based component to a functional component
+const ContactList = props => {
+  return (
+    <ul className="contact-list">
+      {props.noContacts}
```
Comment (canaandavis) [inline src/ContactList.js]: We actually don't need to pass a prop `props.noContacts`, if we want to do something in the component to calculate this we can do it passed on the `props.contacts` array that it receives.

Not the end of the world, but just a thought!

**Pull request closed without merging**